src/System.Private.CoreLib/src/System/Globalization/FormatProvider.DateTimeFormat.cs

```diff
@@ -1,1213 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Diagnostics;
-using System.Text;
-using System.Threading;
-using System.Globalization;
-using System.Collections.Generic;
-using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
-using System.Security;
```

Comment:
we need to return the FormatProvider files. it is a refactoring for the globalization dependency